README.md

```diff
@@ -1,32 +1,48 @@
 # Pylint extension for Visual Studio Code
 
-A Visual Studio Code extension with support for the `pylint` linter. The extension ships with `pylint=3.0.1`.
+A Visual Studio Code extension with support for the Pylint linter. This extension ships with `pylint=3.0.1`.
 
-Note:
+This extension supports all [actively supported versions](https://devguide.python.org/#status-of-python-branches) of the `Python` language (i.e., Python >= 3.8).
 
--   This extension is supported for all [actively supported versions](https://devguide.python.org/#status-of-python-branches) of the `Python` language (i.e., Python >= 3.8).
--   By default, this extension uses the shipped `pylint` version. However, you can use `pylint` from your environment by setting `pylint.importStrategy` to `fromEnvironment`. Alternatively, you can use a custom `pylint` executable by setting `pylint.path`.
--   Minimum supported version of `pylint` is `2.12.2`.
 
-## Usage
+## Usage and Features
 
-Once installed in Visual Studio Code, pylint will be automatically executed when you open a Python file.
+The Pylint extension provides a series of features to help your productivity while working with Python code in Visual Studio Code. Check out the [Settings section](#settings) below for more details on how to customize the extension.
 
-If you want to disable pylint, you can [disable this extension](https://code.visualstudio.com/docs/editor/extension-marketplace#_disable-an-extension) per workspace in Visual Studio Code.
+- **Integrated Linting**: Once this extension is installed in Visual Studio Code, Pylint is automatically executed when you open a Python file, providing immediate feedback on your code quality.
+- **Customizable Pylint Version**: By default, this extension uses the version of Pylint that is shipped with the extension. However, you can configure it to use a different binary installed in your environment through the `pylint.importStrategy` setting, or set it to a custom Pylint executable through the `pylint.path` settings.
+- **Immediate Feedback**: By default, Pylint will update the diagnostics in the editor once you save the file. But you can get immediate feedback on your code quality as you type by enabling the `pylint.lintOnChange` setting.
+- **Mono repo support**: If you are working with a mono repo, you can configure the extension to lint Python files in subfolders of the workspace root folder by setting the `pylint.cwd` setting to `${fileDirname}`.
+- **Customizable Linting Rules**: You can customize the severity of specific Pylint error codes through the `pylint.severity` setting.
+
+> **Note**: The minimum version of Pylint this extension supports is `2.12.2`.
+
+### Disabling Pylint
+If you wish to disable linting with Pylint, you can you can [disable this extension](https://code.visualstudio.com/docs/editor/extension-marketplace#_disable-an-extension) in Visual Studio Code. You can disable it for the current workspace or globally.
```

Comment:
`pylint.ignorePatterns` can be used to skip workspaces in multiroot or mono-repo scenarios.